src/styles/main.css

```diff
@@ -0,0 +1,47 @@
+.container {
```

Comment:
You already have these styles in `container.scss`. No need to duplicate them here. You can also delete `main.css` file, as you have `main.scss`